### DISCUSSION TITLE
Keybindings don't seem to fully work?

### DISCUSSION THREAD

**Discussion by philiprenich** [vuejs/petite-vue#106] [Q&A]
I'm attempting to use `@keyup.up` and others like it. While `@keyup.enter` works fine, others don't. However, `@keyup.arrow-up` is fine. `@keyup.delete` only handles Delete, not Delete and Backspace (need to add the latter).

Here's a simple example. https://codepen.io/philiprenich/pen/qBPZyze

Is this missing or am I misunderstanding something? Thanks!

**Comment by ggedde** [comment]
Same issue. @keyup.esc is not working.
Having to use this instead 
```
@keyup="event.keyCode === 27 ? open = false : null"
```
